### PR TITLE
Update CHANGELOG.md file on release

### PR DESCRIPTION
**Pull request by oskarhane**
Add section for current release in CHANGELOG.md on releases.

### CODE REVIEW

File: .github/workflows/prerelease.yaml [hunk 1]
```diff
@@ -34,10 +34,13 @@ jobs:
       - name: Print varaibles
```
Comment (willtai) [inline .github/workflows/prerelease.yaml:34]: Might as well edit this typo 😃

**Review by willtai — APPROVED** [review]
✅

File: .github/workflows/prerelease.yaml (hunk 1 above)
Comment (oskarhane) [reply .github/workflows/prerelease.yaml:34]: It's fixed in https://github.com/neo4j/neo4j-genai-python/pull/54 so I'd rather keep it like this  here :)

**Comment by oskarhane** [comment]
Look at that @willtai https://github.com/neo4j/neo4j-genai-python/commit/e6c0d4b3547a7ac6da64d931bb7b507705ffdafc, it works.

**Pull request merged**